record.go

```diff
@@ -40,23 +40,36 @@ type Record struct {
 	Payload Data
 }
 
+// Position is a unique identifier for a record. It is the responsibility of the
+// Source to choose and assign record positions, it can freely choose a format
+// that makes sense and contains everything needed to restart a pipeline at a
+// certain position.
 type Position []byte
```

Comment:
Maybe it would make sense, to bring this sentence into the first plan: "contains everything needed to restart a pipeline at a certain position." Positions being unique IDs of records does not necessarily mean that they can be used to restart a pipeline from a position.

Reply:
I spent some time trying to restructure the comment but I can't come up with anything without overcomplicating it. Besides, it is 4 lines, I think a developer should be capable of reading it in one go 😛  There is another comment explaining this more specifically on [`Source.Open`](https://github.com/ConduitIO/conduit-plugin-sdk/blob/db79c240b1faaacc94e9687d5468693deb0e8c07/source.go#L38-L42), that's where the logic is important after all.

That said, I welcome proposals if you already have a better comment in mind!

Reply:
> reading it in one go
Pun intended? : )

But yeah, I definitely agree the comment is good, concise, I was thinking about defining a position primarily as something which is used in a source, to start reading from it. IIUC, that is the most important thing about it. Positions being unique IDs may not be enough.